spec/endtoend/endtoend_spec.rb

```diff
@@ -144,15 +144,7 @@
       expect(@workspace.tsuru_command.stdout).to include query
     end
 
-    it "should be able to connect to the applitation via HTTPS with a valid cert" do
```

Comment:
I see. We thought since it's the only test which  is being skipped right now, it should be at the end of the test suite. We can move it back.